src/config/chainEndpoints.ts

```diff
@@ -23,6 +23,7 @@ export const providerEndpoints: ChainProvider[] = [
     info: 'The main network of the layer 2 scaling blockchain, Plasm Network',
     endpoint: 'wss://rpc.plasmnet.io/',
     favicon: 'favicon.png',
+    isSupportContract: false,
```

Comment:
I suggest using this pattern later when we want to make the application scalable.
https://github.com/polkadot-js/apps/blob/master/packages/apps-routing/src/contracts.ts#L15

This may require a lot of change so we can do this later, but I think it's good to know its pattern.